buku.py

```diff
@@ -146,7 +147,6 @@ def encrypt_file(iterations, dbfile=None):
             from cryptography.hazmat.backends import default_backend
             from cryptography.hazmat.primitives.ciphers import (Cipher, modes,
                                                                 algorithms)
-            from getpass import getpass
```

Comment:
getpass import was intentionally done here. Please keep it this way.

I wanted on demand loading because once the DB file is decrypted it's left as a flat file for regular Buku operations. Didn't want to do an extra import all the time even when executing non-encryption stuff.

Generally either encrypt or decrypt is done in one instance. If you can import it (and other BukuCrypt dependencies) at a class level, I'm fine with it, but not outside the class.

If you need getpass in your test case implementation, please import separately outside.